pkg/cli/admin/catalog/catalog.go

```diff
@@ -0,0 +1,28 @@
+package catalog
+
+import (
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "catalog",
+		Short: "Tools for managing the OpenShift OLM Catalogs",
+		Long: templates.LongDesc(`
+			This tool is used to extract and mirror the contents of catalogs for Operator
+			Lifecycle Manager.
+
+			The subcommands allow you to build catalog images from a source (such as appregistry) 
+			and mirror its content across registries.
+			`),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
`Run:   kcmdutil.DefaultSubCommandRun(streams.ErrOut),` like all the other commands.